src/main/java/edu/kis/powp/command/ComplexCommandBuilder.java

```diff
@@ -0,0 +1,24 @@
+package edu.kis.powp.command;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ComplexCommandBuilder{
+
+    private List<DriverCommand> commands;
+
+    public ComplexCommandBuilder(){
+        commands = new ArrayList<>();
+    }
+
+    public void addCommand(DriverCommand command){
+        commands.add(command);
+    }
+
+
+    public ComplexCommand build(){
+        return new ComplexCommand(commands);
```

Comment:
It would be safer to create a copy of commands (so the builder cannot modify it after build)